packages/mimikatz.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,49 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'mimikatz'
```

Comment:
Be sure to enclose code in a try / catch. For an example see:
- https://github.com/mandiant/VM-Packages/blob/main/packages/ollydbg2.scyllahide.vm/tools/chocolateyinstall.ps1

Reply:
Enclosed in try/catch